doc/whats_new/v0.24.rst

```diff
@@ -149,6 +149,15 @@ Changelog
   :pr:`17233` by :user:`Zolisa Bleki <zoj613>`.
 
 
+:mod:`sklearn.feature_selection`
+................................
+
+- |Enhancement| Added the option for the number of n_features_to_select to be
```

Comment:
There is already a `sklearn.feature_selection` section above that we can move this to.